specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/examples/Pricings/PutPricingByNamePartialSuccess_example.json

```diff
@@ -0,0 +1,60 @@
+{
+  "parameters": {
+    "api-version": "2023-08-01-preview",
+    "scopeId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "pricingName": "CloudPosture",
+    "pricing": {
+      "properties": {
+        "pricingTier": "Standard"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/pricings/CloudPosture",
+        "name": "CloudPosture",
+        "type": "Microsoft.Security/pricings",
+        "properties": {
+          "pricingTier": "Standard",
+          "freeTrialRemainingTime": "PT0S",
+          "enablementTime": "2023-03-01T12:42:42.1921106Z",
+          "extensions": [
+            {
+              "name": "AgentlessVmScanning",
+              "isEnabled": "True",
+              "operationStatus": {
+                "code": "Succeeded",
+                "message": ""
+              }
+            },
+            {
+              "name": "AgentlessDiscoveryForKubernetes",
+              "isEnabled": "True",
+              "operationStatus": {
+                "code": "Succeeded",
+                "message": ""
+              }
+            },
+            {
+              "name": "SensitiveDataDiscovery",
+              "isEnabled": "True",
+              "operationStatus": {
+                "code": "Succeeded",
+                "message": ""
+              }
+            },
+            {
+              "name": "ContainerRegistriesVulnerabilityAssessments",
+              "isEnabled": "True",
+              "operationStatus": {
+                "code": "Failed",
+                "message": "Extension enablemment failed due to missing permissions."
```

Comment:
Minor - let's update all extensions with "failed" status except for this one.
At the moment, Containers VA is the only extension that doesn't require high permissions.
For all the rest - they required the same high permissions, so the 3 of them should fail at the same operation.